### ISSUE TITLE
-S plus vs -S minus shows the same reads

### ISSUE THREAD

**Issue by ahishsujay** [guigolab/ggsashimi#51]
Hi, 

I've been using ggsashimi for a while now and it's a wonderful package!

I needed some clarification on `-s` and `-S` arguments. I went through the options as to what the script says the difference between the two are and also went through a similar question someone asked about this. As far as I understand `-s SENSE` would be if you want to visualize the reads for the sense strand and `-s ANTISENSE` would be if you want to visualize the reads for the antisense strand with respect to RNA-Seq. Meanwhile, I would want to visualize the reads of only for the forward strand (+ strand) in one of the junctions I'd like to view and would not want the reverse strand (- strand) reads. I entered the junction coordinates on IGV, viewed the sashimi plot for the forward and reverse strand and for each of the strands there are different numbers. I expected the same output using ggsashimi and used the `-S plus` once and `-S minus` another time but I get the same junction reads using both options and there is no difference in the outputs between them. 

Here is my script execution incase I'm not making any mistakes and if my understanding is right: 
`~/tools/ggsashimi/sashimi-plot.py -b ../input_bam.tsv -c 12:6448738-6451363 -g annotation_CD27.gtf --shrink --alpha 0.6 --base-size=20 --ann-height=4 --height=3 --width=18 -P ../palette.txt -C 1 -S plus -o sashimi_CD27_onlyJunc_plus`

If needed I can attach a screenshot of what I see for plus and minus strands on IGV and what I see for ggsashimi if I'm not doing something wrong.

Thanks, 
Ahish

**Comment by dgarrimar** [comment]
Hi @ahishsujay, thanks for using ggsashimi! 

I'd like to emphasize that  `-s` refers to the strand specificity of the RNA-seq experiment you are looking at. That means that this option should be selected to match the strand specificity in the experiment. If you are not sure of the strand specificity of your experiment you can check it up using http://rseqc.sourceforge.net/#infer-experiment-py (you can get rseqc from conda [here](https://anaconda.org/bioconda/rseqc)). This is important as, for instance, if your experiments are single-end 'unstranded', then it does not make sense to set `-s` to `SENSE` nor `ANTISENSE`. Can you check the strandness of your data? Besides that, it would be great if you could attach the plots that you get with ggsashimi/igv so that we have extra information.

**Comment by padwalmk** [comment]
Hi, 
I am also facing the same issue, my RNA-SEQ data-set is paired-end with the --rf (MATE2-sense). When I am viewing it in IGV and I set forward strand then it shows the reads from the only forward strand.  However, when I am plotting using ggshashimi and plotting using the plus strand I am getting no splice read, while using the minus strand I am getting reads from both strand. 
I have used the following command:


sashimi-plot.py -b bam_list  -c 1:16513437-16666689 -o MSTRG413.4 -g MSTRG413.1 -s MATE2_SENSE -C 3  -S plus --shrink --fix-y-scale  -M 10 --ann-height 2 --width  15 --base-size 14 -R 300 -F jpeg

Sashimi-plot plus strand 
![MSTRG413 4_+](https://user-images.githubusercontent.com/55156932/107939262-7ef55f00-6fac-11eb-9fcd-ae3475166ebe.jpeg)

Sashimi plot minus-strand
![MSTRG413 3_-](https://user-images.githubusercontent.com/55156932/107939271-80bf2280-6fac-11eb-9762-fa8b90ed0755.jpeg)
 I am also attaching an image from the IGV sashimi plot view. 
IGV Screenshot for forward strand. 
![Screenshot from 2021-02-14 08-17-45](https://user-images.githubusercontent.com/55156932/107939547-e6abaa00-6fac-11eb-9105-c2de070d175a.png)

The number of spliced reads also different.
